eng/pipelines/libraries/build-job.yml

```
@@ -48,10 +48,12 @@ jobs:
 
       ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
         dependsOn:
-        - ${{ if ne(parameters.osGroup, 'WebAssembly') }}:
+        - ${{ if and(ne(parameters.osGroup, 'WebAssembly'), ne(parameters.osGroup, 'iOS')) }}:
```

could you instead use, notin? https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#notin

```suggestion
        - ${{ if notin(parameters.osGroup, 'WebAssembly', 'iOS') }}:
```